ssms/basic_simulators/simulator.py

```diff
@@ -698,6 +699,123 @@ def simulator(
             random_state=random_state,
         )
 
+    if model == "full_SSP": 
+        # Note: here simulate n samples for two congruency condition, thus 2 * n_samples totally
+        # This simulation strategy is consistent with R package flankr. 
+        diff_sigma = kwargs.pop("diff_sigma", 0.1)
```

Comment:
`diff_sigma` here is different from `s` parameter?

This setting will conflict with the `no_noise` option (which can be useful for plotting), because you are overriding choice of `s`. 

Can you propose an alternative?